### ISSUE TITLE
Is this app still supported?

### ISSUE THREAD

**Issue by oivindi** [belangeo/cecilia5#106]
I've been trying to contact the owner through the official Cecilia web page, but no response.

Just wondering about compatibility with Big Sur, and with M1 Macs.

**Comment by DarkAlchy** [comment]
I don't think that it is.

**Comment by maronid** [comment]
There are some issues with BigSur and window display (range sliders) It should not be super hard to fix. I have tried to contact the developer but I also did not get a reply.

**Comment by belangeo** [comment]
Yeah, I don't have much time to spend on Cecilia anymore... I'll see if I can come up with a quick fix for the sliders.

**Comment by belangeo** [comment]
Fix is in git: https://github.com/belangeo/cecilia5/commit/655954e60b6aecc4bfbd06e378aae356b2da6d14

**Comment by maronid** [comment]
Hi Olivier,

Thank you so much for your answer. Cecilia is such a great tool. It would be great if you could now and then maybe just update the very basics.
I wonder if anyone would take the responsibility to maintain it soon.

Thank you for all what you have contributed to the community

Warm Regards
D

1 + 1 = 1
www.maronidis.net <http://www.maronidis.net/>

**Comment by maronid** [comment]
I guess it means that the app has to be built right or I can just replace the resources	in the package content?


1 + 1 = 1
www.maronidis.net <http://www.maronidis.net/>

**Comment by oivindi** [comment]
I just replaced the content in "Resources" with the updated files, seems to work just fine!

Thanks so much for this, Olivier, really appreciated. :)

**Comment by belangeo** [comment]
Yep, this will work as long as the fixes will only concern cecilia's sources. If I change things in pyo (or if there are changes in wxPython, the GUI toolkit), it won't be as simple... 

How I see it, unless someone take over the packaging, is that the best thing to do is to run Cecilia from sources. It goes like this:

1. Install python from python.org
2. Install dependencies:
`pip install --user pyo wxPython numpy`
3. Clone Cecilia's repo:
`git clone https://github.com/belangeo/cecilia5.git`
4. Then, to start the application:
```
cd cecilia5
python3 Cecilia5.py
```

**Comment by DarkAlchy** [comment]
Python is the problem not a solution.  C/C++ as I have had nothing but issues with Python stuff plus it is slower, and overly bloated in comparison to ML/ASM/C/C++.  Python was made to bring an idea to the table fast it was never meant for what so many use it as today.  Like 3d printing was made to bring an idea to the table fast (rapid processing), refine it then use a viable material for the final product which is exactly what Python was originally intended to be for.  Fast to the table, refine it, make sure everything is running right then use a real hardened language for the to market thing.  Most things on Github (these days) seem to skip that last part.

**Comment by maronid** [comment]
I cannot install the dependencies. Getting error messages and the building process freezes. 


1 + 1 = 1
www.maronidis.net <http://www.maronidis.net/>

**Comment by RpxdYTX** [comment]
> Python is the problem not a solution. C/C++ as I have had nothing but issues with Python stuff plus it is slower, and overly bloated in comparison to ML/ASM/C/C++. Python was made to bring an idea to the table fast it was never meant for what so many use it as today. Like 3d printing was made to bring an idea to the table fast (rapid processing), refine it then use a viable material for the final product which is exactly what Python was originally intended to be for. Fast to the table, refine it, make sure everything is running right then use a real hardened language for the to market thing. Most things on Github (these days) seem to skip that last part.

Python is still a language that can totally be used, and while i personally don't, the devs have the choice of using it or not.

**Comment by DarkAlchy** [comment]
> > Python is the problem not a solution. C/C++ as I have had nothing but issues with Python stuff plus it is slower, and overly bloated in comparison to ML/ASM/C/C++. Python was made to bring an idea to the table fast it was never meant for what so many use it as today. Like 3d printing was made to bring an idea to the table fast (rapid processing), refine it then use a viable material for the final product which is exactly what Python was originally intended to be for. Fast to the table, refine it, make sure everything is running right then use a real hardened language for the to market thing. Most things on Github (these days) seem to skip that last part.
> 
> Python is still a language that can totally be used, and while i personally don't, the devs have the choice of using it or not.

Doesn't make it a wise choice, or the best choice.  They can use toilet tissue if they wanted, or hammer, and a chisel on a big stone tablet but that doesn't make that a viable choice.

**Comment by RpxdYTX** [comment]
> 

Comparing the complexity and maintainability of Python and C++, the most "unviable" between the two is C++. Python's runtime speed isn't a problem if the C++ bottlenecks are taken in account

**Comment by DarkAlchy** [comment]
> > 
> 
> Comparing the complexity and maintainability of Python and C++, the most "unviable" between the two is C++. Python's runtime speed isn't a problem if the C++ bottlenecks are taken in account

Only to the ignorant.  A good dev (rare these days especially with the ones I have seen on Github side projects) maintains the code and you get a binary, and/or the source to compile yourself.  There is this thing called versioning that helps with stupid issues.

Python is a scripted language so any damn monkey with a keyboard can fiddle with it while C/C++ et al. are real compilation languages so requires far more brain cells, in that we agree as what you call "unviable" I call a lazy dumb ass approach on the dev's part(s), and the end user being even dumber who can't figure out how to compile the stuff.

**Comment by oivindi** [comment]
Please. This is not a Usenet discussion forum.
